alibi_detect/saving/_pytorch/saving.py

```diff
@@ -129,3 +130,20 @@ def save_embedding_config(embed: TransformerEmbedding,
     embed.model.save_pretrained(filepath)
 
     return cfg_embed
+
+
+def save_device_config(device: TorchDeviceType):
+    """
+
+    Parameters
+    ----------
+    device
+        Device type used. The default tries to use the GPU and falls back on CPU if needed.
+            Can be specified by passing either ``'cuda'``, ``'gpu'``, ``'cpu'`` or an instance of
+            ``torch.device``.
```

Comment:
Unnecessary indent?

Reply:
Docstring also seems slightly inaccurate? Maybe just something like `Torch device to be serialised`?